### DISCUSSION TITLE
Is there any way I can set the look value id of same item that is being pushed with Batch Operation?

### DISCUSSION THREAD

**Discussion by gopaldahal** [pnp/pnpcore#497] [General]
I have an item on the list. Some of the columns within the items point to the self. 
Case of Self look up.

Thank you

Sharing is Caring!

**Comment by jansenbe** [comment]
Need to investigate that

**Reply by jansenbe** [reply]
Rest batches are processed in order (unlike Microsoft Graph batches)...so if you create item1 and item2 in a batch and item2 has a lookup column to item1 then I think that should work. Best to give it a try to confirm
